### DISCUSSION TITLE
Welcome to custom-sonos-card Discussions!

### DISCUSSION THREAD

**Discussion by punxaphil** [punxaphil/custom-sonos-card#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by richardabramson1** [comment]
Hey thanks for this feature! I am a noob and I cannot figure out how to get this to work. It won't show any of my Sonos devices. Can you help me out here?

Thanks.

**Comment by psjnr84** [comment]
Hi Team,

Could I potentially find out if you could add an option to the new queue functionality screen to clear the queue on the selected player?

I’m currently having to do this in a room by room basis with a custom button under each instance of the player. It would be great if your code could invoke this dynamically when you are on each Sonos device.

would you know if this is possible?

**Reply by punxaphil** [reply]
Please create an issue for this
